src/test/java/com/google/cloud/tools/maven/it/RunMojoIntegrationTest.java

```diff
@@ -86,6 +97,34 @@ public void run() {
     assertEquals("Hello from the App Engine Standard project.", urlContent.toString());
     verifier.verifyErrorFreeLog();
     verifier.verifyTextInLog("Dev App Server is now running");
+    verifier.verifyTextInLog("Module instance " + expectedModuleName + " is running");
+  }
+
+  @SuppressWarnings("unused") // used for testRun()
```

Comment:
JUnitParameters uses the convention that if the test method is called fooBar, then the method providing the parameters is called parametersForFooBar (a convention that can be overridden, but that wouldn't change the fact that Eclipse marks it as unused).